sdk/tests/conformance/extensions/ext-blend-func-extended.html

```diff
@@ -0,0 +1,573 @@
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFunc sfactor`);
+        gl.blendFunc(gl.ONE, extFuncs[func]);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFunc dfactor`);
+        gl.blendFuncSeparate(extFuncs[func], gl.ONE, gl.ONE, gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFuncSeparate srcRGB`);
+        gl.blendFuncSeparate(gl.ONE, extFuncs[func], gl.ONE, gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFuncSeparate dstRGB`);
+        gl.blendFuncSeparate(gl.ONE, gl.ONE, extFuncs[func], gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFuncSeparate srcAlpha`);
+        gl.blendFuncSeparate(gl.ONE, gl.ONE, gl.ONE, extFuncs[func]);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFuncSeparate dstAlpha`);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be no errors");
+    }
+
+    const dbi = gl.getExtension("OES_draw_buffers_indexed");
+    if (!dbi) return;
+
+    debug("");
+    debug("Testing indexed SRC1 blend funcs without the extension");
+    for (const func in extFuncs) {
+        dbi.blendFunciOES(0, extFuncs[func], gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFunciOES src`);
+        dbi.blendFunciOES(0, gl.ONE, extFuncs[func]);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFunciOES dst`);
+        dbi.blendFuncSeparateiOES(0, extFuncs[func], gl.ONE, gl.ONE, gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFuncSeparateiOES srcRGB`);
+        dbi.blendFuncSeparateiOES(0, gl.ONE, extFuncs[func], gl.ONE, gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFuncSeparateiOES dstRGB`);
+        dbi.blendFuncSeparateiOES(0, gl.ONE, gl.ONE, extFuncs[func], gl.ONE);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFuncSeparateiOES srcAlpha`);
+        dbi.blendFuncSeparateiOES(0, gl.ONE, gl.ONE, gl.ONE, extFuncs[func]);
+        wtu.glErrorShouldBe(gl, gl.INVALID_ENUM, `${func} not accepted as blendFuncSeparateiOES dstAlpha`);
+        wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be no errors");
+    }
+}
+
+function runEnumTests() {
+    debug("");
+    debug("Testing enums");
+    shouldBe("ext.SRC1_COLOR_EXT", "0x88F9");
+    shouldBe("ext.SRC1_ALPHA_EXT", "0x8589");
+    shouldBe("ext.ONE_MINUS_SRC1_COLOR_EXT", "0x88FA");
+    shouldBe("ext.ONE_MINUS_SRC1_ALPHA_EXT", "0x88FB");
+    shouldBe("ext.MAX_DUAL_SOURCE_DRAW_BUFFERS_EXT", "0x88FC");
+}
+
+function runQueryTests() {
+    debug("");
+    debug("Testing getParameter");
+    shouldBeGreaterThanOrEqual("gl.getParameter(ext.MAX_DUAL_SOURCE_DRAW_BUFFERS_EXT)", "1");
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be no errors");
+
+    if (contextVersion == 1) {
```

Comment:
I find it confusing when a test under `conformance/` changes behavior when run as a WebGL 2.0 test. Would you consider refactoring this into a .js file which is included both by `conformance/extensions/ext-blend-func-extended.html` and `conformance2/extensions/ext-blend-func-extended.html`, so each one runs with the particular context version? `conformance/extensions/00_test_list.txt` would mark that version of the test expiring in version `1.9.9` so it doesn't run in the 2.0 conformance suite. There are several examples of similarly factored tests in `sdk/tests/js/tests/`.

Reply:
Done.